tests/utils/myconfig.js

```diff
@@ -96,20 +96,20 @@ function initRS(servers, port, rsSettings, keyFile, root, user) {
     retval.initiate(cfg);
 
     if (keyFile) {
-        admindb = retval.getMaster().getDB("admin");
+        admindb = retval.getPrimary().getDB("admin");
```

Comment:
This will break test execution for 2.4.x, since `getPrimary()` isn't defined. I think we can mitigate that with adding the following to the top of `myconfig.js`:

```
if (ReplSetTest.getPrimary === undefined) {
    ReplSetTest.getPrimary = ReplSetTest.getMaster;
}
```